content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst

```diff
@@ -0,0 +1,356 @@
+contains examples of how to generate a barcode for common items in a warehouse.
+
+Product + quantity + lot
+------------------------
+
+To build a GS1 barcode for a box that contains a product, number of units in it, and the lot number,
+the following barcode patterns are used:
+
++------------+--------------------------+------+----------------------------------+------------------------------------------+
+|    Name    |        Rule Name         | A.I. |       Barcode Pattern            |              Field in Odoo               |
++============+==========================+======+==================================+==========================================+
+| Product    | Global Trade Item Number | 01   | (01)(\\d{14})                    | :guilabel:`Barcode` field on product form|
+|            | (GTIN)                   |      |                                  |                                          |
++------------+--------------------------+------+----------------------------------+------------------------------------------+
+| Quantity   | Variable count of items  | 30   | (30)(\\d{0,8})                   | :guilabel:`Units` field on transfer form |
++------------+--------------------------+------+----------------------------------+------------------------------------------+
+| Lot Number | Batch or lot number      | 10   | (10)([!"%-/0-9:-?A-Z_a-z]{0,20}) | :guilabel:`Lot` on Detailed Operations   |
+|            |                          |      |                                  | pop-up                                   |
++------------+--------------------------+------+----------------------------------+------------------------------------------+
+
+.. _barcode/operations/lot-setup:
+
+Configuration
+~~~~~~~~~~~~~
+
+To track products using lots, first enable the :ref:`Lots and Serial Numbers
+<inventory/management/track_products_by_lots>` feature. To do so, navigate to
+:menuselection:`Inventory app --> Configuration --> Settings`. Next, under the
+:guilabel:`Traceability` heading, check the box for :guilabel:`Lots & Serial Numbers`.
+
+Then, set up the product barcode by navigating to the intended product form in
+:menuselection:`Inventory app --> Products --> Products` and selecting the product. On the product
+form, click :guilabel:`Edit`. Then, in the :guilabel:`General Information` tab, fill in the
+:guilabel:`Barcode` field with the 14-digit `Global Trade Item Number (GTIN)
+<https://www.gs1.org/standards/get-barcodes>`_, which is a universal and unique identifying number
+from GS1.
+
+.. important::
+   On the product form, omit the :abbr:`A.I. (application identifier)` `01` for GTIN product barcode
+   pattern, as it is only used to encode multiple barcodes into a single barcode that contains
+   detailed information about the package contents.
+
+.. example::
+   To create a barcode for the product, `Fuji Apple`, enter the 14-digit GTIN `12345678901231` in
```

Comment:
this isn't helpful because this is demonstrates exactly what *not* to do 😓 

GTINs in particular follow very specific formatting rules, yes, however you can't just make them up (even if you follow the rules for how they're made, such as making sure the character count is right)— these numbers represent uniquely recognizable products in the global supply-chain and therefore have unique IDs. Here's [a table, for example,](https://resources.markon.com/product-catalog/apples-fuji) that shows a bunch of *actual* Fuji Apple GTIN/UPC values.

Would expand the example much more in the next iteration...as it reads right now it communicates "you can just make numbers up and put them here on the form"